fuzzers/060-bram-cascades/Makefile

```diff
@@ -0,0 +1,38 @@
+MAKETODO_FLAGS=--re "BRAM_.\.BRAM_(?!LOGIC_OUTS).*"
+export FUZDIR=$(shell pwd)
+PIPLIST_TCL=$(FUZDIR)/bram_pip_list.tcl
+PIP_TYPE?=bram_pips_int
+SEG_TYPE?=bram
+N = 50
+SEGMATCH_FLAGS=-m 20 -M 45
+include ../pip_loop.mk
+#
+# Specimens from current run must complete, but previous iterations may exist
+database: $(SPECIMENS_OK)
+	${XRAY_SEGMATCH} $(SEGMATCH_FLAGS) -o build/segbits_bram_l.rdb $(shell find build -name segdata_bram_l.txt)
+	${XRAY_SEGMATCH} $(SEGMATCH_FLAGS) -o build/segbits_bram_r.rdb $(shell find build -name segdata_bram_r.txt)
+
+	# Keep a copy to track iter progress
+	# Also is pre-fixup, which drops and converts
+	cp build/segbits_bram_l.rdb build/$(ITER)/segbits_bram_l.rdb
+	cp build/segbits_bram_r.rdb build/$(ITER)/segbits_bram_r.rdb
+	${XRAY_DBFIXUP} --db-root build \
```

Comment:
Are the bits in BRAM_L different than BRAM_R? Traditionally pip fuzzers separate l vs r for solving, but configuration fuzzers solve them together. I'm not sure why this is the case though, this convention was inherited from Clifford

Reply:
I answered this question in another message, but the bits are the same between BRAM_L and BRAM_R, however the pip names are different.  E.g


26_56  26_57  26_59 ,BRAM.BRAM_ADDRBWRADDRL1.BRAM_IMUX_ADDRBWRADDRL1
vs
26_56  26_57  26_59 ,BRAM.BRAM_ADDRBWRADDRL1.BRAM_R_IMUX_ADDRBWRADDRL1

With some string mangling I should be able to unify the fuzzing which will improve the numbers of samples required to solve all pips.

Reply:
Refactored generate.py to mangle the pips, and added unmanagle to regenerated the L and R versions.